src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -1978,17 +2189,24 @@ boolean isPostGISDataStore(DataStore dataStore) {
     }
 
     /*
-     * computes the lat/lon bounding box from the native bounding box and srs, optionally overriding
+     * Computes the lat/lon bounding box from the native bounding box and srs, optionally overriding
      * the value already set.
+     *
+     * If the does not contain a geometry (such as a csv file processing lat/long into a point)
+     * this activity may not have required information to work with.
      */
     boolean computeLatLonBoundingBox(ImportTask task, boolean force) throws Exception {
         ResourceInfo r = task.getLayer().getResource();
         if (force || r.getLatLonBoundingBox() == null && r.getNativeBoundingBox() != null) {
             CoordinateReferenceSystem nativeCRS = CRS.decode(r.getSRS());
             ReferencedEnvelope nativeBbox =
                     new ReferencedEnvelope(r.getNativeBoundingBox(), nativeCRS);
-            r.setLatLonBoundingBox(nativeBbox.transform(CRS.decode("EPSG:4326"), true));
-            return true;
+
+            if (!nativeBbox.isEmpty()) {
+                r.setLatLonBoundingBox(nativeBbox.transform(CRS.decode("EPSG:4326"), true));
+                return true;
+            }
+            LOGGER.fine("Import '" + r.getName() + "' native bounding box is empty");
```

Comment:
Hum... ok... but wouldn't it be better to set an empty box for the lat/lon bbox too instead?

Reply:
Good idea will do.

If you check out the doc update I was testing with a CSV file that required a transform to get a geometry, so It was ending up here.